.php-cs-fixer.dist.php

```diff
@@ -10,11 +10,22 @@
     ->setRules([
         '@Symfony' => true,
         'array_syntax' => ['syntax' => 'short'],
+        'array_indentation' => true,
         'self_static_accessor' => true,
+        'global_namespace_import' => [
```

Comment:
I usually do `false, false, true`, since everything apart of classes do not need the leading `\` and it results in the smallest possible fine.

But that is my personal preference.